app/models/work.rb

```diff
@@ -31,6 +31,10 @@ class Work < ApplicationRecord
       transition first_draft: :deposited, version_draft: :deposited
     end
 
+    event :submit_for_review do
+      transition first_draft: :pending_approval
```

Comment:
`pending_approval` or `pending_review` or :man_shrugging: ?

Reply:
"Pending approval" was the language used in the mockups, so I used that.

Reply:
Good call!